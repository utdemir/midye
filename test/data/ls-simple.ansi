a
[0m[01;34mb[0m
c
[40;31;01mfoobar[0m
