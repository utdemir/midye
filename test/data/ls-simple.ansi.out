a
TSControlCharacter CC_LF
TSTermSequenceCSI (TSC_SGR [0])
TSTermSequenceCSI (TSC_SGR [1,34])
b
TSTermSequenceCSI (TSC_SGR [0])
TSControlCharacter CC_LF
c
TSControlCharacter CC_LF
TSTermSequenceCSI (TSC_SGR [40,31,1])
foobar
TSTermSequenceCSI (TSC_SGR [0])
TSControlCharacter CC_LF

